bolt/spark/chunk.py

```diff
@@ -136,80 +147,124 @@ def _unchunk(v):
 
         switch = self.switch
         rdd = self._rdd.map(switch).groupByKey().mapValues(_unchunk)
-        return BoltArraySpark(rdd, shape=self.shape, split=self._split)
 
-    def move(self, kaxes=(), vaxes=()):
-        """
-        Move some of the dimensions in the values into the keys.
+        if array_equal(self.vshape, [1]):
+            rdd = rdd.mapValues(lambda v: squeeze(v))
+            newshape = self.shape[:-1]
+        else:
+            newshape = self.shape
+
+        return BoltArraySpark(rdd, shape=newshape, split=self._split)
 
-        Along with chunking, this is required as part of the swap method
-        on the BoltArraySpark, see there for further details.
+    def keystovalues(self, axes, size=None):
```

Comment:
thoughts on renaming these `keys_to_values` and `values_to_keys`? i think i slightly prefer that.